console/src/main/java/org/georchestra/console/ws/backoffice/users/UsersController.java

```diff
@@ -604,6 +606,14 @@ private Account modifyAccount(Account account, ServletInputStream inputStream) t
 				account.setShadowExpire((new SimpleDateFormat("yyyy-MM-dd")).parse(shadowExpire));
 		}
 
+		String privacyPolicyAgreementDate = RequestUtil.getFieldValue(json, UserSchema.PRIVACY_POLICY_AGREEMENT_DATE_KEY);
```

Comment:
Maybe should accoun dao impl choose what to do with empty date... In my option opinion, field should be let blank in dto. Here logic is if "", then set it to null, and at dao side if null delete it.

Reply:
Yes, it would be better

Reply:
Hmmm, I think we should keep it this way: if the field is not present in JSON, the Account field is let unmodified. But if the JSON contains the field, specifically set to empty value, I imagine it's logical to understand that the Account field should be emptied (obviously, this behavior should be written in the API doc... to be written in the same style than https://app.swaggerhub.com/apis-docs/groldan/extractorapp/1.0.0).
If we don't do that way, we remove the possibility to the user to empty that field.